tests/test_pfas_feature_extraction.py

```
@@ -65,15 +61,14 @@
     )
-    from moml.utils import extract_fluorine_count
     from moml.utils import add_fluorinated_group_counts
     from moml.core import MolecularGraphProcessor
     from moml.utils import visualize_molecular_graph
 
     IMPORTS_SUCCESSFUL = True
     print("Successfully imported moml modules!")
-except ImportError as e:
-    print(f"Failed to import required moml modules: {e}")
+except ImportError:
+    print(f"Failed to import required moml modules")
```

Ruff (0.12.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)